index.html

```diff
@@ -110,10 +128,13 @@
         game.restart(true)
       })
     </script>
+
+    <div id="outcome" style="display: block;"></div>
     <br>
     <span class="btn customize">
       CUSTOMIZE
       <code>new Game(cols, rows, bombs, [emptyemoji, bombemoji, flagemoji, starteremoji], twemojiOrNot)</code>
     </a>
```

Comment:
^ heh this can be deleted. The `<span>` was originally `<a>` which was why it isn't closed. :laughing: